examples/lusid/orders/order-blocks/create-custom-property-on-order-block/4-set-new-property-to-block.sql

```diff
@@ -0,0 +1,24 @@
+-- define contingent id
+
+@@contingent_id = SELECT '670';
+
+-- Loading in block data from Excel spreadsheet
```

Comment:
Don't think this should use the spreadsheet - it's confusing

Reply:
Removed spreadsheet access